.goreleaser.yml

```diff
@@ -24,6 +24,7 @@ builds:
     main: .
     goos:
       - linux
+      - darwin
```

Comment:
need to change the line 22
from:
```
binary: incert-linux-{{ .Arch }}
```

to 
```
binary: incert-{{ .OS }}-{{ .Arch }}
```

Reply:
Good catch!